client/homebrew/editor/metadataEditor/metadataEditor.less

```diff
@@ -43,22 +61,33 @@
 					background-color: #777;
 				}
 			}
-			.thumbnail-preview{
-				position    : relative;
-				width       : 80px;
-				height      : min-content;
-				border      : 2px solid white;
-				margin-left : 5px;
-				max-height  : 115px;
+		}
+
+		&.description {
+			flex: 1;
+			textarea.value {
+				resize      : none;
+				height      : auto;
+				font-family : 'Open Sans', sans-serif;
+				font-size   : 0.8em;
 			}
 		}
 	}
-	.description.field textarea.value{
-		resize      : none;
-		height      : 5em;
-		font-family : 'Open Sans', sans-serif;
-		font-size   : 0.8em;
+
+
+	.thumbnail-preview {
+		position: relative;
+		justify-self: center;
+		width: 80px;
+		height: min-content;
+		border: 2px solid white;
+		flex: 1 1;
+		max-height: 115px;
+		aspect-ratio: 1 / 1;
+		object-fit: contain;
+		background-color: cadetblue;
```

Comment:
Something weird happens with this background color when the panel is very narrow. This container stretches out on the left and right and looks a bit odd. Any way we can just maintain the aspect ratio?

![image](https://user-images.githubusercontent.com/5878534/190886134-b1f75a38-5363-4ed9-ac07-fcb5b68ce5d5.png)

Reply:
oh i actually kinda like it.   I don't like to mix left-aligned content (the labels and input fields) with centered content (the thumbnail).    It seems odd and empty.

This is entirely an aesthetic opinion, so I will go whichever way the wind blows.

Reply:
@jeddai @G-Ambatte @ericscheid opinions?

Reply:
I think I'd actually argue for a middle ground of keep it left-aligned, but the alignment should be off of the leftmost position of the field. So the second part of the image where we see the left hand of the thumbnail lines up with the left hand of the input above it.

Reply:
To what Jeddai says--- I think that might be tough because the thumbnail preview is on the same 'row' as the description and URL fields and just flexes to that second row.  I'm not sure it'd be possible to have both:

- right-situated of the description/URL fields at wider screen size, and
- a blank space to the left when flexed to the bottom.

I could change the color of that background square or reduce it's saturation to make it less 'eye-popping'.  Though I like the bright splashes of color that are present elsewhere in the UI.  Additionally maybe removing the white border helps?

This PR is not terribly important so it can simmer for a bit until more opinions come in.

Reply:
@Gazook89  I have two main concerns:

- The white border seems weird where it encompasses all this blank space. It's especially weird when the image is not square, so the border doesn't quite touch the image. Maybe removing the border would help as you say.

![image](https://user-images.githubusercontent.com/5878534/190944414-95cf39a0-88a2-41ab-92c4-7f2be0fee493.png)

- The turquoise color clashes very badly with some other colors a user might choose:

![image](https://user-images.githubusercontent.com/5878534/190944457-33412c60-fe0b-4e2f-8dc2-f6b198107ec1.png)

Maybe just another neutral gray?

![image](https://user-images.githubusercontent.com/5878534/190946774-c4b835c8-0578-44a0-8b76-ac78c61812c8.png)

I don't know. Everything else about this PR is fine; just something about the styling here is not quite jiving with me.